### PR TITLE
remove Python2 support

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request removes Python 2 support.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ notifications:
   email: false
 
 python:
-- '2.7'
 - '3.6'
 before_install:
 - sudo apt-get -y install gcc libattr1-dev
```
Comment (lachmanfrantisek) [inline .travis.yml]: I think we are not using travis at all. We can remove the whole file.

File: release-conf.yaml [hunk 2]
```diff
@@ -1,7 +1,3 @@
-# list of major python versions that release-bot will build separate wheels for
-python_versions:
-  - 3
```
Comment (jpopelka) [inline release-conf.yaml]: We need this until new-release bot (with https://github.com/user-cont/release-bot/pull/147) is released

File: release-conf.yaml [hunk 3]
```diff
@@ -1,7 +1,3 @@
...
-
```
Comment (lachmanfrantisek) [inline release-conf.yaml]: Please, leave that here until the new release of release bot..;-)

File: README.md [hunk 4]
```diff
@@ -37,6 +37,7 @@ have to compile it yourself when getting it from PyPI.
 $ pip3 install --user colin
 ```
 
+`colin` does NOT support python 2.
```
Comment (lachmanfrantisek) [inline README.md]: I think we can say, that the "support" is for python >=3.6.

File: release-conf.yaml (hunk 3 above)
Comment (phracek) [reply release-conf.yaml]: We would like to get rid of python-2 support in this sprint. We will release release-bot w/o python2 support soon.

Comment (lachmanfrantisek) [reply release-conf.yaml]: Then LGTM.

**Pull request merged**